### ISSUE TITLE
Text on Path shows properly in browser, not in pdf

### ISSUE THREAD

**Issue by c-murray** [yWorks/svg2pdf.js#152]
**Describe the bug**

**To Reproduce**
See this [playground](http://raw.githack.com/yWorks/svg2pdf.js/master/?svg=%253Csvg%2520viewBox%253D%25220%25200%25204032%25204032%2522%2520xmlns%253D%2522http%253A%252F%252Fwww.w3.org%252F2000%252Fsvg%2522%2520data-name%253D%2522feature-row__image%2520product-featured-media%2520lazyautosizes%2520ls-is-cached%2520lazyloaded%2522%2520id%253D%2522FeaturedMedia-meow-meow-custom%2522%253E%250A%2520%2520%253Cstyle%253E%2523doit%2520%257B%250A%2520%2520%2520%2520font-size%253A%2520520px%253B%250A%257D%253C%252Fstyle%253E%250A%2520%2520%253Cdefs%253E%250A%2520%2520%2520%2520%2520%2520%253ClinearGradient%2520gradientTransform%253D%2522rotate%2875%29%2522%2520y2%253D%25220%2525%2522%2520x2%253D%2522100%2525%2522%2520y1%253D%25220%2525%2522%2520x1%253D%25220%2525%2522%2520id%253D%2522grad1%2522%2520xmlns%253D%2522http%253A%252F%252Fwww.w3.org%252F2000%252Fsvg%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cstop%2520style%253D%2522stop-color%253Argb%28150%252C150%252C150%29%253Bstop-opacity%253A1%2522%2520offset%253D%25220%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cstop%2520style%253D%2522stop-color%253Argb%28220%252C220%252C220%29%253Bstop-opacity%253A1%2522%2520offset%253D%252210%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%2509%253Cstop%2520style%253D%2522stop-color%253Argb%28150%252C150%252C160%29%253Bstop-opacity%253A1%2522%2520offset%253D%252220%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cstop%2520style%253D%2522stop-color%253Argb%28150%252C150%252C160%29%253Bstop-opacity%253A1%2522%2520offset%253D%252250%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cstop%2520style%253D%2522stop-color%253Argb%28150%252C150%252C160%29%253Bstop-opacity%253A1%2522%2520offset%253D%252270%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cstop%2520style%253D%2522stop-color%253Argb%28220%252C220%252C220%29%253Bstop-opacity%253A1%2522%2520offset%253D%252290%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%253Cstop%2520style%253D%2522stop-color%253Argb%28150%252C150%252C150%29%253Bstop-opacity%253A1%2522%2520offset%253D%2522100%2525%2522%253E%253C%252Fstop%253E%250A%2520%2520%2520%2520%2520%2520%253C%252FlinearGradient%253E%250A%2520%2520%2520%2520%253C%252Fdefs%253E%250A%2520%2520%253Ctitle%253EMONOGRAM%2520E%2520W%2520INNER%2520BOX%253C%252Ftitle%253E%250A%2520%2520%253Cg%2520data-name%253D%2522Layer%25202%2522%2520id%253D%2522Layer_2%2522%253E%250A%250A%2520%2520%2520%2520%253Cpath%2520x1%253D%2522271.86%2522%2520d%253D%2522M271.86%252C2363L3783.73%252C2363%2522%2520class%253D%2522cls-1%2522%2520id%253D%2522curvee%2522%253E%253C%252Fpath%253E%250A%250A%2520%2520%253C%252Fg%253E%250A%2520%2520%253Ctext%2520class%253D%2522cls-1%2522%2520width%253D%2522400%2522%2520id%253D%2522dertext%2522%2520fill%253D%2522%25239695a0%2522%253E%253CtextPath%2520text-anchor%253D%2522middle%2522%2520startOffset%253D%252250%2525%2522%2520xlink%253Ahref%253D%2522%2523curvee%2522%2520id%253D%2522doit%2522%253E%250AAVERY%253C%252FtextPath%253E%253C%252Ftext%253E%253C%252Fsvg%253E).

**Expected behavior**
I would have expected the PDF to look like so. 
This is in accordance to the following SVG spec.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by HackbrettXXX** [comment]
Yes, text paths are not supported. See also #82. We would be very happy about a pull request, though ;)

**Comment by HackbrettXXX** [comment]
Specification reference: https://www.w3.org/TR/SVG2/text.html#TextLayoutPath

For an implementation we might "steal" some code from [Canvg](https://github.com/canvg/canvg/blob/master/src/Document/TextPathElement.ts).

**Comment by listopadiya** [comment]
Seems to be challenging but interesting task, I can take it if you don't mind.

**Comment by HackbrettXXX** [comment]
Sure, go ahead ;)
